net/mwan3/files/usr/share/rpcd/ucode/mwan3

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+import { popen, readfile } from 'fs';
+import { cursor } from 'uci';
+
+let ubus = require('ubus').connect();
+
+function get_str_raw(iface, property) {
+    return readfile(sprintf('/var/run/mwan3track/%s/%s', iface, property));
```

Comment:
I get the sneaking suspicion that this might already be available via `procd` and something like `ubus call service list`. Altho this way may be 'less expensive' to call.

Reply:
I don't understand your comment here.
mwan3track store its state in /var/run/mwan3track/, here we read those small files to export all the info via rpcd/ubus

Reply:
I was thinking you could already read it from ubus, like `ubus call network.interface.mwan3 status` or something.

Reply:
I already call ubus twice, many of those property are mwan3track specific and must be read from `/var/run/mwan3track/`